This fails for us as we try to build with the system's rapidjson.
Since we're not the CI of the upstream project, this is useless to us anyway

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8f6f143ad..851b48725 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1024,8 +1024,6 @@ if(MBGL_WITH_CORE_ONLY)
     return()
 endif()
 
-include(${PROJECT_SOURCE_DIR}/scripts/license.cmake)
-
 if(NOT EXISTS ${PROJECT_SOURCE_DIR}/internal/internal.cmake)
     set(MBGL_PUBLIC_BUILD TRUE)
 endif()
